QuasselDroid/src/main/java/com/iskrembilen/quasseldroid/Quasseldroid.java

```diff
@@ -25,6 +27,7 @@ public void onCreate() {
         //Load current theme
         ThemeUtil.initTheme(this);
         BusProvider.getInstance().register(this);
+        context = this;
```

Comment:
Gussing these are changes that are left over from refactored code. It is never used